src/lib/Sendbird.tsx

```diff
@@ -310,110 +311,112 @@ const SendbirdSDK = ({
-      value={{
-        stores: {
-          sdkStore,
-          userStore,
-          appInfoStore,
-        },
-        dispatchers: {
-          sdkDispatcher,
-          userDispatcher,
-          appInfoDispatcher,
-          reconnect,
-        },
-        config: {
-          disableMarkAsDelivered,
-          renderUserProfile,
-          onUserProfileMessage,
-          allowProfileEdit,
-          isOnline,
-          userId,
-          appId,
-          accessToken,
-          theme: currentTheme,
-          setCurrentTheme,
-          setCurrenttheme: setCurrentTheme, // deprecated: typo
-          isMultipleFilesMessageEnabled,
-          uikitUploadSizeLimit: uploadSizeLimit ?? DEFAULT_UPLOAD_SIZE_LIMIT,
-          uikitMultipleFilesMessageLimit,
-          userListQuery,
-          logger,
-          pubSub,
-          imageCompression: {
-            compressionRate: 0.7,
-            outputFormat: 'preserve',
-            ...imageCompression,
-          },
-          voiceRecord: {
-            maxRecordingTime: voiceRecord?.maxRecordingTime ?? VOICE_RECORDER_DEFAULT_MAX,
-            minRecordingTime: voiceRecord?.minRecordingTime ?? VOICE_RECORDER_DEFAULT_MIN,
-          },
-          userMention: {
-            maxMentionCount: userMention?.maxMentionCount || 10,
-            maxSuggestionCount: userMention?.maxSuggestionCount || 15,
-          },
-          markAsReadScheduler,
-          markAsDeliveredScheduler,
-          // Remote configs set from dashboard by UIKit feature configuration
-          common: configs.common,
-          groupChannel: {
-            enableOgtag: sdkInitialized && configsWithAppAttr(sdk).groupChannel.channel.enableOgtag,
-            enableTypingIndicator: configs.groupChannel.channel.enableTypingIndicator,
-            enableReactions: sdkInitialized && configsWithAppAttr(sdk).groupChannel.channel.enableReactions,
-            enableReactionsSupergroup: sdkInitialized && configsWithAppAttr(sdk).groupChannel.channel.enableReactionsSupergroup,
-            enableMention: configs.groupChannel.channel.enableMention,
-            replyType: configs.groupChannel.channel.replyType,
-            threadReplySelectType: configs.groupChannel.channel.threadReplySelectType,
-            enableVoiceMessage: configs.groupChannel.channel.enableVoiceMessage,
-            enableDocument: configs.groupChannel.channel.input.enableDocument,
-            typingIndicatorTypes: configs.groupChannel.channel.typingIndicatorTypes,
-            enableFeedback: configs.groupChannel.channel.enableFeedback,
-            enableSuggestedReplies: configs.groupChannel.channel.enableSuggestedReplies,
-            showSuggestedRepliesFor: configs.groupChannel.channel.showSuggestedRepliesFor,
-            suggestedRepliesDirection: configs.groupChannel.channel.suggestedRepliesDirection,
-            enableMarkdownForUserMessage: configs.groupChannel.channel.enableMarkdownForUserMessage,
+    <div id="sendbird-provider-wrapper">
```

Comment:
여긴 provider 의 역할만 하고  div 가 붙지 않는게 맞는거같아요. SendbirdProvider 를 쓰는 고객들도 갑자기 이 div 가 추가되면 div 가 가진 기본 스타일 설정들로 인해 원치않게 전체 스타일이 깨질 수도 있답니당

Reply:
How about now? I think we need to wrap all our components with this div tag to accurately determine the area covered by the UIKit, so that we can precisely calculate the position of the menu components.

Reply:
Hmm still, there might be an issue with adding an additional div element here since div has its default style settings like `display: block`.